app/views/movies/show.html.erb

```diff
@@ -52,6 +51,11 @@
   </aside>
 </section>
 <div id="review-container">
+  <div id="form-error"></div>
+  <div id="review-form-container"></div>
+</div>
+<div id="reviews">
+  <%= render partial: 'shared/reviews', movie: @movie %>
```

Comment:
No need for this to be inside `shared` dir. Just add the partial inside `reviews`, `reviews/_reviews.html.erb`.
shared is for shared views which are accessed all over the place, for example flash_messages. Reviews are belonging to the reviews :)